react/src/vite-env.d.ts

```diff
@@ -1 +1,26 @@
 /// <reference types="vite/client" />
+
+interface Env {
+  designSafeUrl: string;
+  backend: string;
+  apiUrl: string;
+  portalUrl: string;
```

Comment:
i think these should be more explicit as both are used as apis and we got so many portals out there. how about:

apiUrl -> geoApiUrl
portalUrl -> designSafePortalUrl